OutOfSchool/OutOfSchool.AuthCommon/Services/ProviderAdminService.cs

```diff
@@ -125,17 +118,14 @@ public class ProviderAdminService : IProviderAdminService
                         userId,
                         result.ErrorMessages());
```

Comment:
I believe `result.ErrorMessages()` here should be `roleAssignResult.ErrorMessages()`